src/openrct2/cheats.c

```diff
@@ -21,6 +21,7 @@
 #include "localisation/date.h"
 #include "management/finance.h"
 #include "network/network.h"
+#include "util\util.h"
```

Comment:
I did so. I rebased it as well, though come to think of it, I probably should have waited until we found two more overflows. my bad